src/Runtime/omp/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ if(TARGET omp)
 
   add_custom_target(libOMomp
```

Comment:
For consistency, this `libOMomp` and the one in `src/Compiler/CMakeLists.txt` should also be changed to `ompruntime`.